diff --git a/crates/context_aware_config/src/db/schema.rs b/crates/context_aware_config/src/db/schema.rs
index 15c2eee..25c8088 100644
--- a/crates/context_aware_config/src/db/schema.rs
+++ b/crates/context_aware_config/src/db/schema.rs
@@ -2,13 +2,13 @@
 
 diesel::table! {
     config_versions (id) {
         id -> Int8,
         config -> Json,
         config_hash -> Text,
-        tags -> Nullable<Array<Nullable<Varchar>>>,
+        tags -> Nullable<Array<Varchar>>,
         created_at -> Timestamp,
     }
 }
diesel::table! {
     contexts (id) {
