diff --git a/crates/context_aware_config/src/db/schema.rs b/crates/context_aware_config/src/db/schema.rs
index 15c2eee..25c8088 100644
--- a/crates/context_aware_config/src/db/schema.rs
+++ b/crates/context_aware_config/src/db/schema.rs
@@ -2,13 +2,13 @@
 
 diesel::table! {
     config_versions (id) {
         id -> Int8,
         config -> Json,
         config_hash -> Text,
-        tags -> Nullable<Array<Nullable<Varchar>>>,
+        tags -> Nullable<Array<Varchar>>,
         created_at -> Timestamp,
     }
 }
 
diesel::table! {
    webhooks (name) {
        name -> Text,
        description -> Text,
        enabled -> Bool,
        url -> Text,
        method -> Text,
        version -> Text,
        custom_headers -> Nullable<Json>,
-       events -> Array<Nullable<Varchar>>,
+       events -> Array<Varchar>,
        max_retries -> Int4,
        last_triggered_at -> Nullable<Timestamp>,
        created_by -> Text,
        created_at -> Timestamp,
        last_modified_by -> Text,
        last_modified_at -> Timestamp,
    }
}

diesel::table! {
     contexts (id) {
