diff --git a/crates/experimentation-platform/src/db/schema.rs b/crates/experimentation-platform/src/db/schema.rs
index ef6ee2d..8199439 100644
--- a/crates/experimentation-platform/src/db/schema.rs
+++ b/crates/experimentation-platform/src/db/schema.rs
@@ -9,10 +9,6 @@ pub mod cac_v1 {
         #[derive(diesel::sql_types::SqlType)]
         #[diesel(postgres_type(name = "experiment_status_type", schema = "cac_v1"))]
         pub struct ExperimentStatusType;
-
-        #[derive(diesel::sql_types::SqlType)]
-        #[diesel(postgres_type(name = "not_null_text", schema = "cac_v1"))]
-        pub struct NotNullText;
     }
 
     diesel::table! {
@@ -54,7 +50,6 @@ pub mod cac_v1 {
 
     diesel::table! {
         use diesel::sql_types::*;
-        use super::sql_types::NotNullText;
         use super::sql_types::ExperimentStatusType;
 
         cac_v1.experiments (id) {
@@ -63,7 +58,7 @@ pub mod cac_v1 {
             created_by -> Text,
             last_modified -> Timestamptz,
             name -> Text,
-            override_keys -> Array<Nullable<NotNullText>>,
+            override_keys -> Array<Text>,
             status -> ExperimentStatusType,
             traffic_percentage -> Int4,
             context -> Json,
