diff --git a/crates/experimentation-platform/src/db/schema.rs b/crates/experimentation-platform/src/db/schema.rs
index 692c718..46a426c 100644
--- a/crates/experimentation-platform/src/db/schema.rs
+++ b/crates/experimentation-platform/src/db/schema.rs
@@ -6,6 +6,2 @@ pub mod sql_types {
     pub struct ExperimentStatusType;
-
-    #[derive(diesel::sql_types::SqlType)]
-    #[diesel(postgres_type(name = "not_null_text"))]
-    pub struct NotNullText;
 }
@@ -183,3 +179,2 @@ diesel::table! {
     use diesel::sql_types::*;
-    use super::sql_types::NotNullText;
     use super::sql_types::ExperimentStatusType;
@@ -192,3 +187,3 @@ diesel::table! {
         name -> Text,
-        override_keys -> Array<Nullable<NotNullText>>,
+        override_keys -> Array<Text>,
         status -> ExperimentStatusType,
@@ -217,2 +212,2 @@ diesel::allow_tables_to_appear_in_same_query!(
     experiments,
-);
+);
\ No newline at end of file
