diff --git a/crates/experimentation-platform/src/db/schema.rs b/crates/experimentation-platform/src/db/schema.rs
index fc346c1..887ba9e 100644
--- a/crates/experimentation-platform/src/db/schema.rs
+++ b/crates/experimentation-platform/src/db/schema.rs
@@ -13,13 +13,13 @@ diesel::table! {
     experiments (id) {
         id -> Int8,
         created_at -> Timestamptz,
         created_by -> Text,
         last_modified -> Nullable<Timestamptz>,
         name -> Text,
-        override_keys -> Array<Nullable<Text>>,
+        override_keys -> Array<Text>,
         status -> ExperimentStatusType,
         traffic_percentage -> Int4,
         context -> Json,
         variants -> Json,
     }
 }
