diff --git a/crates/superposition_types/src/database/schema.rs b/crates/superposition_types/src/database/schema.rs
index 3d66e604..c4cb4f9c 100644
--- a/crates/superposition_types/src/database/schema.rs
+++ b/crates/superposition_types/src/database/schema.rs
@@ -5,9 +5,6 @@ pub mod sql_types {
     #[diesel(postgres_type(name = "experiment_status_type"))]
     pub struct ExperimentStatusType;

-    #[derive(diesel::sql_types::SqlType)]
-    #[diesel(postgres_type(name = "not_null_text"))]
-    pub struct NotNullText;
 }

 diesel::table! {
@@ -15,7 +12,7 @@ diesel::table! {
         id -> Int8,
         config -> Json,
         config_hash -> Text,
-        tags -> Nullable<Array<Nullable<Varchar>>>,
+        tags -> Nullable<Array<Varchar>>,
         created_at -> Timestamp,
     }
 }
@@ -614,7 +611,6 @@ diesel::table! {

 diesel::table! {
     use diesel::sql_types::*;
-    use super::sql_types::NotNullText;
     use super::sql_types::ExperimentStatusType;

     experiments (id) {
@@ -623,7 +619,7 @@ diesel::table! {
         created_by -> Text,
         last_modified -> Timestamptz,
         name -> Text,
-        override_keys -> Array<Nullable<NotNullText>>,
+        override_keys -> Array<Text>,
         status -> ExperimentStatusType,
         traffic_percentage -> Int4,
         context -> Json,
diff --git a/crates/superposition_types/src/database/superposition_schema.rs b/crates/superposition_types/src/database/superposition_schema.rs
index bd298000..4772a860 100644
--- a/crates/superposition_types/src/database/superposition_schema.rs
+++ b/crates/superposition_types/src/database/superposition_schema.rs
@@ -22,7 +22,7 @@ pub mod superposition {
             last_modified_by -> Text,
             last_modified_at -> Timestamp,
             created_at -> Timestamp,
-            mandatory_dimensions -> Nullable<Array<Nullable<Text>>>,
+            mandatory_dimensions -> Nullable<Array<Text>>,
         }
     }
 }
