diff --git a/crates/superposition_types/src/database/schema.rs b/crates/superposition_types/src/database/schema.rs
index 4946d3ad..d38a2cc2 100644
--- a/crates/superposition_types/src/database/schema.rs
+++ b/crates/superposition_types/src/database/schema.rs
@@ -1,24 +1,20 @@
 // @generated automatically by Diesel CLI.

 pub mod sql_types {
     #[derive(diesel::sql_types::SqlType)]
     #[diesel(postgres_type(name = "experiment_status_type"))]
     pub struct ExperimentStatusType;
-
-    #[derive(diesel::sql_types::SqlType)]
-    #[diesel(postgres_type(name = "not_null_text"))]
-    pub struct NotNullText;
 }

 diesel::table! {
     config_versions (id) {
         id -> Int8,
         config -> Json,
         config_hash -> Text,
-        tags -> Nullable<Array<Nullable<Varchar>>>,
+        tags -> Nullable<Array<Varchar>>,
         created_at -> Timestamp,
         change_reason -> Text,
     }
 }

 diesel::table! {
@@ -618,22 +614,21 @@ diesel::table! {
         query -> Text,
     }
 }

 diesel::table! {
     use diesel::sql_types::*;
-    use super::sql_types::NotNullText;
     use super::sql_types::ExperimentStatusType;

     experiments (id) {
         id -> Int8,
         created_at -> Timestamptz,
         created_by -> Text,
         last_modified -> Timestamptz,
         name -> Text,
-        override_keys -> Array<Nullable<NotNullText>>,
+        override_keys -> Array<Text>,
         status -> ExperimentStatusType,
         traffic_percentage -> Int4,
         context -> Json,
         variants -> Json,
         last_modified_by -> Text,
         chosen_variant -> Nullable<Text>,
