diff --git a/crates/superposition_types/src/database/schema.rs b/crates/superposition_types/src/database/schema.rs
index 8359708e..c1c0b01e 100644
--- a/crates/superposition_types/src/database/schema.rs
+++ b/crates/superposition_types/src/database/schema.rs
@@ -5,10 +5,6 @@ pub mod sql_types {
     #[diesel(postgres_type(name = "experiment_status_type"))]
     pub struct ExperimentStatusType;
 
-    #[derive(diesel::sql_types::SqlType)]
-    #[diesel(postgres_type(name = "not_null_text"))]
-    pub struct NotNullText;
-
     #[derive(diesel::sql_types::SqlType)]
     #[diesel(postgres_type(name = "workspace_status"))]
     pub struct WorkspaceStatus;
@@ -38,7 +34,7 @@ diesel::table! {
         id -> Int8,
         config -> Json,
         config_hash -> Text,
-        tags -> Nullable<Array<Nullable<Varchar>>>,
+        tags -> Nullable<Array<Varchar>>,
         created_at -> Timestamp,
     }
 }
@@ -637,7 +633,6 @@ diesel::table! {
 
 diesel::table! {
     use diesel::sql_types::*;
-    use super::sql_types::NotNullText;
     use super::sql_types::ExperimentStatusType;
 
     experiments (id) {
@@ -646,7 +641,7 @@ diesel::table! {
         created_by -> Text,
         last_modified -> Timestamptz,
         name -> Text,
-        override_keys -> Array<Nullable<NotNullText>>,
+        override_keys -> Array<Text>,
         status -> ExperimentStatusType,
         traffic_percentage -> Int4,
         context -> Json,
