diff --git a/crates/superposition_types/src/database/superposition_schema.rs b/crates/superposition_types/src/database/superposition_schema.rs
index eed781eb..251e7fa8 100644
--- a/crates/superposition_types/src/database/superposition_schema.rs
+++ b/crates/superposition_types/src/database/superposition_schema.rs
@@ -50,13 +50,13 @@ pub mod superposition {
             workspace_status -> WorkspaceStatus,
             workspace_admin_email -> Text,
             created_by -> Text,
             last_modified_by -> Text,
             last_modified_at -> Timestamp,
             created_at -> Timestamp,
-            mandatory_dimensions -> Nullable<Array<Nullable<Text>>>,
+            mandatory_dimensions -> Nullable<Array<Text>>,
         }
     }
 
     diesel::joinable!(workspaces -> organisations (organisation_id));
 
     diesel::allow_tables_to_appear_in_same_query!(
diff --git a/crates/superposition_types/src/database/superposition_schema.rs b/crates/superposition_types/src/database/superposition_schema.rs
index f92a20e6..01329567 100644
--- a/crates/superposition_types/src/database/superposition_schema.rs
+++ b/crates/superposition_types/src/database/superposition_schema.rs
@@ -30,8 +30,8 @@ pub mod superposition {
             status -> OrgStatus,
             #[max_length = 100]
             sector -> Nullable<Varchar>,
-            created_at -> Timestamp,
-            updated_at -> Timestamp,
+            created_at -> Timestamptz,
+            updated_at -> Timestamptz,
             updated_by -> Text,
         }
     }
@@ -51,8 +51,8 @@ pub mod superposition {
             workspace_admin_email -> Text,
             created_by -> Text,
             last_modified_by -> Text,
-            last_modified_at -> Timestamp,
-            created_at -> Timestamp,
+            last_modified_at -> Timestamptz,
+            created_at -> Timestamptz,
             mandatory_dimensions -> Nullable<Array<Text>>,
         }
     }
